addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -465,6 +464,31 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasDescription() {
+            return Boolean(
+                this.thread &&
+                (this.thread.description ||
+                this.thread.isChannelDescriptionChangeable)
+            );
+        }
```

Comment:
This field name is also wrong according to the compute.

Actually having a description is unrelated to whether the description is changeable.

Reply:
@seb-odoo Can you please suggest the field name?

Reply:
Literally what it contains `hasDescriptionOrDescriptionPlaceholder`, or maybe `hasDescriptionArea` if we want a more generic name.